xtable-core/src/test/java/org/apache/xtable/delta/TestDeltaSync.java

```diff
@@ -431,6 +433,46 @@ public void testGetTargetCommitIdentifierWithNullSourceIdentifier() throws Excep
     assertFalse(unmappedTargetId.isPresent());
   }
 
+  @Test
+  public void testTimestampNtz() {
+    InternalSchema schema1 = getInternalSchemaWithTimestampNtz();
+    List<InternalField> fields2 = new ArrayList<>(schema1.getFields());
+    fields2.add(
```

Comment:
Have you tested whether the write works if the initial schema contains the timestamp_ntz?

Reply:
Yes in this test, schema1 contains ntz column and the second commit is adding a nullable float column.